packages/react-docs/src/components/navigation/navigation.js

```diff
@@ -42,8 +42,11 @@ class Navigation extends React.Component {
     const { searchValue } = this.state;
     const searchRE = new RegExp(searchValue, 'i');
 
-    const filteredComponentRoutes = componentRoutes.filter(c =>
-      searchRE.test(c.label)
+    const hiddenComponents = ['AlertAction', 'AlertBody', 'AlertIcon'];
```

Comment:
this should no longer be needed since routes are generated base off of files under `pages/{(components|layouts)}/{page}.js` The hidden components listed would not be shown either way.